requirements/base.in

```diff
@@ -0,0 +1,11 @@
+# Core requirements for using this package
+-c constraints.txt
+
+lxml
+bleach
+django
+oauthlib
+mako
+lazy
+XBlock
+xblock-utils>=v1.0.0
```

Comment:
Why not move this constraint to `constraints.txt`?

Reply:
I remove this constraint and tests are working fine.